CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Upcoming
+
+* Remove @warn from link helpers because they are annoying
```

Comment:
By "they are annoying", do you mean "they are printed even when the extension isn't used"?